### PR TITLE
Adding osmosis-test-5 path info and noble global fees info.

### PR DESCRIPTION
**Pull request by danbryan**
add ibc atom to fee_tokens

### CODE REVIEW

**Comment by JeremyParish69** [comment]
I assume we're just waiting for this to take effect. Can we please also have IBC'd ATOM added to the assetlist as well since it's a fee token?

**Comment by danbryan** [comment]
Hey @JeremyParish69 I added atom to the assetlist, and we will mark this ready for review once noble has completed the fee upgrade.

**Comment by danbryan** [comment]
Hey @JeremyParish69 this is ready to be reviewed.  Please let me know if i missed anything!

File: testnets/_IBC/nobletestnet-osmosis5testnet.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "noble",
+    "client_id": "07-tendermint-8",
+    "connection_id": "connection-7"
+  },
+  "chain_2": {
+    "chain_name": "osmosis5",
```
Comment (JeremyParish69) [inline testnets/_IBC/nobletestnet-osmosis5testnet.json]: these chain names should be 'nobletestnet' and we'll be using 'osmosistestnet5'

**Pull request merged**